CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## v5.0.0 (2020-07-xx)
+
+[Full Changelog]
+
+**Breaking changes:**
+
+- drop support for deprecated puppet/staging module
+
+**Breaking changes:**
```

Comment:
why do we have two titles with breaking changes?

Reply:
Good point - removed